src/common/json-schema-input/array-input.js

```diff
@@ -122,7 +135,7 @@ export default class ArrayInput extends Component {
       !propsEqual(
         this.props,
         props,
-        ['depth', 'label', 'required', 'schema', 'uiSchema', 'value']
+        [ 'depth', 'disabled', 'label', 'required', 'schema', 'uiSchema' ]
```

Comment:
IMHO it would be better to update `value` from the props instead of via a setter, but if it works for now, don't touch it.

Anyway it makes sense to only handle one of the way to set it.